msm-sitemap.php

```diff
@@ -599,7 +626,7 @@ public static function get_last_modified_posts() {
 	public static function get_post_dates( $posts ) {
 		$dates = array();
 		foreach ( $posts as $post ) {
-			$dates[] = date( 'Y-m-d', strtotime( $post->post_date ) );
+			$dates[] = date( 'Y-m-d', strtotime( $post['post_date'] ) );
```

Comment:
Lol. I see that's fixed here. I'm still unclear as to why we're changing the behaviour :)

Reply:
I like this, perhaps we should mention it in the commit message

Reply:
Cool, I'll be sure to include that in the merge commit.